File: Example.Data.Contract/Model/CustomerDto.cs
namespace Example.Data.Contract.Model
{
    using System;
    using System.Collections.Generic;

    using Covis.Data.Common;

    public class CustomerDto : IModelEntity
    {
        public CustomerDto()
        {
            this.Contacts = new HashSet<ContactDto>();
        }

        public long Id { get; set; }

        public int EdvNr { get; set; }

        public string Firma1 { get; set; }

        public string Firma2 { get; set; }

        public string Street { get; set; }

        public string Ort { get; set; }

        public int ContactCount { get; set; }

        public IEnumerable<ContactDto> Contacts { get; set; }
    }
}

File: Example.Angular/ScriptsApp/model/CustomerDto.ts

 

    export class CustomerDto implements IModel{ 
        type: string;
		constructor() {
			this.type = "Example.Data.Contract.Model.CustomerDto,Example.Data.Contract";
			
                    
        }      
        
		
		id: number;
		edvNr: number;
		firma1: string;
		firma2: string;
		street: string;
		ort: string;
		contacts: ContactDto[];
    }
